### PR TITLE
ReadMe edit

### PR DESCRIPTION
**Pull request by JamesFrierson1**
Minor grammar and punctuation changes

### CODE REVIEW

**Review by KevinRaney — APPROVED** [review]
Looks good, thanks for contributing @JamesFrierson1!

**Comment by KevinRaney** [comment]
I redirected this to the develop branch, your documentation changes will be included in the next release.

**Comment by JamesFrierson1** [comment]
Sounds great! Thanks for letting me contribute! Let me know if you need another review once the Merge Calendar List section is updated 👍

**Pull request merged**
